servant/src/Servant/API/ContentTypes.hs

```diff
@@ -371,28 +364,9 @@ instance NFData NoContent
 --------------------------------------------------------------------------
 -- * MimeUnrender Instances
 
```

Comment:
```suggestion
-- | Deprecated: since aeson version 0.9 `eitherDecode` has lenient behavior.
--
 eitherDecodeLenient :: FromJSON a => ByteString -> Either String a
 eitherDecodeLenient = eitherDecode
 {-# DEPRECATED eitherDecodeLenient "use eitherDecode instead" #-}
```